contents/blog/how-to-pivot.md

```diff
@@ -0,0 +1,142 @@
+
+We confused a lot of people with this idea, because we ourselves were confused with whom we were targeting.
+
+We got 15 VPs of sales to agree to trying this out, sent them 
+
+With hindsight, it was way overpowered for tiny teams and we'd only have had a great fit for huge ones with a lot of data. The only people interested in smaller teams were, to use [Crossing the Chasm's framework](https://en.wikipedia.org/wiki/Crossing_the_Chasm), innovators, which doesn't represent most of the market. And doing _only_ sales into big companies would be really tough - slower sales, a better product required, and a longer cycle time so reduced learning.
+
+### 2. CRM with predictive analytics
+
+We kept getting asked for the above in the CRM versus standalone, so we integrated it, but then we saw very early users just using it as their CRM - they didn't need the complexity of Salesforce, so we just thought we'd provide a CRM for tiny companies.
+
+This was a poor decision - there are many "lightweight CRMs" out there, so getting attention was very hard, and predictive analytics make more sense for those with more data. This was around the time that [Superhuman](https://superhuman.com/) was getting pretty popular; we got overexcited, and kept using words like "blazing", "gorgeous", "brilliant".
+
+We didn't think through who we were building for. The market is also pretty saturated with noise, so we could have focused more on our differentiation - a product may have made more sense than a platform here. We couldn't get people to agree to try it. From memory we got a single customer for $20/month, who then didn't actually pay the invoice.
+
+### 3. 1:1 tool with predictive analytics
+
+We felt that the actual problem we were solving was still prioritization of where to focus time.
+
+We were both big believers in [Andrew Grove](https://en.wikipedia.org/wiki/Andrew_Grove)'s [High Output Management](https://www.amazon.com/High-Output-Management-Andrew-Grove/dp/0679762884/ref=sr_1_1?dchild=1&keywords=high+output+management&qid=1610712757&s=books&sr=1-1).
+
+By this stage, we were thinking we just wanted to work with people that would at least try our stuff.
+
+### 4. technical debt monitoring tool using surveys after each pull request
+
+We finally started getting a little clearer over those we were build
+
+It turns out everyone has problems with technical debt, but solving it involves changing how teams prioritize.
+
+We just weren't the right people to run this business. Although Tim had struggled with technical debt first hand, neither of us had struggled with actually solving it. If one of us had managed an engineering team before, we'd have perhaps been better placed to understand this area.
```

Comment:
might want to have a think as to how this portrayal lines up with Tim as a CTO from the reader's perspective